hir/src/component/mod.rs

```diff
@@ -86,11 +84,14 @@ impl formatter::PrettyPrint for ComponentImport {
         };
         let name = match self {
             ComponentImport::CanonAbiImport(import) => import.interface_function.to_string(),
-            ComponentImport::MidenAbiImport(import) => import.function_id.to_string(),
+            ComponentImport::MidenAbiImport(_import) => "()".to_string(),
+        };
+        let import = match self {
+            ComponentImport::CanonAbiImport(_) => const_text("import"),
+            ComponentImport::MidenAbiImport(_) => const_text("import_miden"),
```

Comment:
I wonder if it might not be better to do like we do with function calling conventions, and use a marker attribute for the ABI of an import, i.e. `(import (abi canon)` or `(import (abi miden)`, and so on.

Reply:
Nice! Done. I do not like `import_miden`. I just put it there as a sort of placeholder to differentiate between the types of imports.